libs/ui-watt/src/lib/components/shell/nav-list/watt-nav-list.component.html

```diff
@@ -0,0 +1,32 @@
+<!--
+@license
+Copyright 2020 Energinet DataHub A/S
+
+Licensed under the Apache License, Version 2.0 (the "License2");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<ng-container *ngIf="expandable; else navListTemplate">
+  <mat-expansion-panel
+    wattExpandOnActiveLink
+    [wattNavListItemComponents]="navListItemComponents"
+    class="mat-elevation-z0"
```

Comment:
Where is this class used? :)

Reply:
This is part of Material. A so called "elevation helper". More info here https://material.angular.io/guide/elevation